duecredit/io.py

```diff
@@ -154,6 +154,29 @@ def _model_citations(self, tags=None):
                 target_dict = packages
             target_dict['citations'][path].append(citation)
             target_dict['entry_keys'][path].append(entry_key)
+
+        # now we need to filter out the packages that don't have modules
+        # or objects cited
+        cited_packages = list(packages['citations'].keys())
+        cited_modules = list(modules['citations'].keys())
+        cited_objects = list(objects['citations'].keys())
+
+        for cited_package in cited_packages:
```

Comment:
you could just do `for cited_package in packages['citations']` here (no need for the variable)